tests/core/pyspec/eth2spec/test/electra/sanity/test_slots.py

```diff
@@ -25,6 +25,26 @@ def run_epoch_processing(spec, state, pending_deposits=None, pending_consolidati
     assert state.pending_consolidations == []
 
 
+@with_electra_and_later
+@spec_state_test
+def test_pending_deposit_extra_gwei(spec, state):
+    # Deposit where the amount has a little extra gwei
+    index = len(state.validators)
+    deposit = prepare_pending_deposit(
+        spec,
+        validator_index=index,
+        # The deposit amount includes some gwei (the +1 at the end)
+        amount=spec.MIN_ACTIVATION_BALANCE + 1,
```

Comment:
Same comment as for the deposit request test, shouldn't this be `amount = spec.EFFECTIVE_BALANCE_INCREMENT + spec.Gwei(1)`